wordpress/wp-content/plugins/cds-base/languages/cds-snc-fr_CA.po

```diff
@@ -821,92 +792,90 @@ msgstr "Alerte"
 #: build/index.js:3368 build/index.js:3432 build/index.js:3323
 #: build/index.js:3377
 msgid "Summary"
-msgstr "le sommaire"
+msgstr "Sommaire"
 
 #: build/index.js:3409 build/index.js:3354
 msgid "Expander"
-msgstr "Expanseur"
+msgstr "Décompresseur"
 
 #: build/index.js:3497 build/index.js:3432
 msgid "Show “Read More”"
-msgstr ""
+msgstr "Afficher « En savoir plus »"
 
 #: build/index.js:3604 build/index.js:3529
-#, fuzzy
 msgid " Tables exist from deleted sites"
-msgstr "Des tables existent à partir de collections supprimées"
+msgstr "Des tables existent à partir de sites supprimés"
 
 #: build/index.js:3787 build/index.js:3702
 msgid "List Name"
-msgstr "Liste de noms"
+msgstr "Nom de la liste"
 
 #: build/index.js:3789 build/index.js:3704
 msgid "Subscriber Count"
-msgstr "Nombre d’abonnés"
+msgstr "Nombre d’abonné·e·s"
 
 #: build/index.js:3828 build/index.js:3743
 msgid "Send Template"
-msgstr "Envoyer le gabarit"
+msgstr "Envoyer le modèle"
 
 #: build/index.js:3912 build/index.js:3817
 msgid "This list has "
-msgstr "Cette liste a"
+msgstr "Cette liste comprend"
 
 #: build/index.js:3914 build/index.js:3819
 msgid " subscribers.  "
-msgstr "les abonnés."
+msgstr "abonné·e·s."
 
 #: build/index.js:3915 build/index.js:3820
-msgid "You won't be able to revert this"
-msgstr "Vous ne pourrez pas revenir en arrière"
+msgid "You won’t be able to revert this"
+msgstr "Vous ne pouvez annuler l’opération."
```

Comment:
If we update the english string, we should also update it in the code.

Reply:
this would have been updated by the merge-po script, so it must have been updated in the code